privaterelay/middleware.py

```diff
@@ -75,3 +75,14 @@ def __call__(self, request):
         ])
 
         return response
+
+class StoreFirstVisit:
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+        response = self.get_response(request)
+        first_visit = request.COOKIES.get("first_visit")
+        if (first_visit is None and not request.user.is_anonymous):
```

Comment:
We want to make it so the NPS survey doesn't show up until 3 days after their first *SIGNED IN* visit?

Reply:
That was my assumption, but let me double-check.

Reply:
> The intention of showing the NPS banner would be that the user has had time to play with the experience and get more used to it, so if it's not too much scope to do "three days after their first visit if they're signed in"

So this is indeed the intended behaviour :)